backend/app/assets/stylesheets/spree/backend/shared/_tables.scss

```diff
@@ -4,27 +4,9 @@ table {
   border-spacing: 0;
 
   th, td {
-    border-right: 1px solid $color-border;
-    border-bottom: 1px solid $color-border;
-    vertical-align: middle;
+    vertical-align: top;
```

Comment:
I agree that the large line-height is not ideal, but I also think that the vertical alignment of single line rows should be corrected. What makes me thinking is that the default tables in Bootstraps documentation are all correctly aligned. 🤔

I will have a second look into this problem